isso/css/isso.css

```diff
@@ -58,10 +58,14 @@
 }
 .isso-comment > .isso-text-wrapper {
     display: block;
+    padding: 0.95em;
 }
 .isso-comment .isso-follow-up {
     padding-left: calc(7% + 20px);
 }
+.isso-comment.isso-page-author > .isso-text-wrapper {
+    background-color: #bae0ea;
```

Comment:
How did you arrive at that specific color? Will it also look good on dark background?

Reply:
I used a color picker to get roughly the same hue as the link color (on the reply button), and then I lightened and desaturated it.

As for it working on a dark background, it doesn't really fit. However, it looked to me that the rest of the CSS doesn't really plan for a dark background by default either. The background color of the postbox fields and the preview field are set to be very light colors which doesn't fit in on a dark background. Since the text color isn't overridden inside the preview box, it becomes white text on a light background if the outer site is set to have white text on black background. Additionally, the text color when hovering over links in (such as the reply button) is very dark, making it hard to read. A lot of these colors need to be overridden to work well on a dark background.

![reply text hovering on black background](https://user-images.githubusercontent.com/17971474/165407732-d95c5d7e-31ca-4d4a-9c7b-c7e53dc835bd.png)

![white text on light background with preview](https://user-images.githubusercontent.com/17971474/165407789-bebaa1c0-ec5b-4984-a410-423f2d1e0091.png)

Reply:
To be honest, maybe don't include any style changes in this PR then and let users decide.

For instance, I'm using this and would have to tweak my CSS to override the default styling that this PR introduces:
![Screenshot from 2022-04-27 01-38-55](https://user-images.githubusercontent.com/10212877/165410002-98018d4e-3720-4616-8f01-0e7b7878d06b.png)

Reply:
Also, there should be some kind of textual indication for blind/impaired users instead of a purely visual one.
![Screenshot from 2022-04-27 01-41-56](https://user-images.githubusercontent.com/10212877/165410242-c6637e2e-81f9-44d5-a94d-612adbfee992.png)

The `(Author)` text should then be part of `isso/js/app/i18n/en.js` as e.g.
`"postbox-page-author": "(Author)" // or "(Site owner)"`

Reply:
> To be honest, maybe don't include any style changes in this PR then and let users decide.
> 
> For instance, I'm using this and would have to tweak my CSS to override the default styling that this PR introduces: ![Screenshot from 2022-04-27 01-38-55](https://user-images.githubusercontent.com/10212877/165410002-98018d4e-3720-4616-8f01-0e7b7878d06b.png)

I think the commits I just pushed a few minutes ago fixes that due to a [small refactor I did](https://github.com/posativ/isso/compare/0b64ba1866d24b66b0feee2126a016cbac2969b2..a85c85ea159db5f585f6bd9d1e4b08a5e7370ab2?diff=split#diff-c20d9e95fdd0eb015d78774a734e4b2b708656bb788869a2d46140d8d7f152b1).

And I guess I can just take those colors out and instead put them somewhere in the docs anyways.

Reply:
I hope you're not waiting on feedback from my end.

For now, let's leave out the background color changes (also, those might clash with the highlight-linked-comment PR?).

To gather screenshots, you can use the pre-packaged demo site. Run `isso -c share/isso-dev.cfg run` and visit `localhost:8080/demo`. Edit `isso/demo/index.html` to add the author badge `data-isso-` attribute.

Reply:
> I hope you're not waiting on feedback from my end.
> 
> For now, let's leave out the background color changes (also, those might clash with the highlight-linked-comment PR?).
> 
> To gather screenshots, you can use the pre-packaged demo site. Run `isso -c share/isso-dev.cfg run` and visit `localhost:8080/demo`. Edit `isso/demo/index.html` to add the author badge `data-isso-` attribute.

Yeah I wasn't waiting for anything, I've just been putting it off because of other projects I'm working on, and school. I can probably finish this tomorrow though, if not today.